### PR TITLE
fix angular integration

### PR DESCRIPTION
**Pull request by franzbecker**

### CODE REVIEW

File: src/assets/main.js [hunk 1]
```diff
@@ -13,17 +9,37 @@ require.config({
...
-            serviceUrl: 'http://localhost:8080/xtext-service',
-            parent: parent,
-            resourceId: resourceId,
-            syntaxDefinition: "xtext-resources/generated/mode-mydsl",
-            enableSaveAction: true
-        });
-    });
+    console.log(`createXtextEditor(${parent}, ${resourceId})`);
+    require(
+        ["jquery", "ace/ace", "xtext/xtext-ace", "xtext/services/XtextService"],
```
Comment (gunther-bachmann) [inline src/assets/main.js]: we probably need only "xtext/xtext-ace" as required dependency here

File: src/assets/main.js [hunk 2]
```diff
@@ -13,17 +9,37 @@ require.config({
     }
 });
 
+/* Setup ajax / xtext services. */
+require(
+    ["jquery", "ace/ace", "xtext/xtext-ace", "xtext/services/XtextService"],
```
Comment (gunther-bachmann) [inline src/assets/main.js]: we probably don't need "xtext/xtext-ace" as dependency here

File: src/assets/main.js [hunk 3]
```diff
@@ -1,10 +1,6 @@
-let baseUrl = window.location.pathname;
-let fileIndex = baseUrl.indexOf("index.html");
-if (fileIndex > 0) {
-    baseUrl = baseUrl.slice(0, fileIndex);
-}
+let baseUrl = window.location.origin;
```
Comment (gunther-bachmann) [inline src/assets/main.js]: origin returns only protocol, server + port, right? does this prevent us from serving from some deeper url (e.g. https://server:port/some/magic/url/test-editor-web/...)? Do we ever have the need for that? I don't know ...

Comment (franzbecker) [inline src/assets/main.js]: I think it's okay for now. On the other hand if we have a page such as https://origin/some/other/nested/page.html the assets should still be at https://origin/assets so then `window.location.origin` is the right path

**Pull request merged**